src/services/query.js

```diff
@@ -539,6 +545,17 @@ ngeo.Query.prototype.doGetFeatureInfoRequests_ = function(
     wmsGetFeatureInfoUrl =
         goog.uri.utils.setParam(wmsGetFeatureInfoUrl, 'QUERY_LAYERS', lyrStr);
 
+    // add dimensions values
+    var dimensions = items[0].source.dimensions;
+    if (dimensions) {
+      for (var key in dimensions) {
+        var value = this.dimensions[key];
+        if (value !== undefined) {
+          wmsGetFeatureInfoUrl = goog.uri.utils.setParam(wmsGetFeatureInfoUrl, key, value);
+        }
```

Comment:
if the value is undefined and it's defined in `items[0].source.dimensions` shouldn't we get the value from `items[0].source.dimensions`?